### PR TITLE
Docs: getting started SEO changes.

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: getting started SEO changes.

---

### Additional information

_Includes code snippets license key call._

_Should not be kept alive long due to discrepancies and sync errors that will arise over time._

### CODE REVIEW

File: docs/getting-started/setup/editor-lifecycle.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 category: setup
 meta-title: Editor lifecycle | CKEditor 5 documentation
+meta-description:  Handling the editor lifecycle. With examples.
```
Comment (gorzelinski) [inline docs/getting-started/setup/editor-lifecycle.md:4]:
```suggestion
meta-description: Handling the editor lifecycle. With examples.
```

File: docs/getting-started/setup/editor-types.md [hunk 2]
```diff
@@ -66,7 +67,7 @@ See an {@link examples/builds/balloon-block-editor example of the balloon block
 
 The Decoupled editor is named for its unique structure, where the toolbar and editing area are separate elements. This design allows for greater flexibility and customization, making it suitable for a wide range of applications beyond just classic WYSIWYG editing.
 
-The most popular use case for the Decoupled editor is the “document editor”, similar to large editing packages such as Google Docs or Microsoft Word. It works best for creating documents, which are usually later printed or exported to PDF files.
+The most popular use case for the Decoupled editor is the “document editor” similar to large editing packages such as Google Docs or Microsoft Word. It works best for creating documents, which are usually later printed or exported to PDF files.
```
Comment (gorzelinski) [inline docs/getting-started/setup/editor-types.md:70]: I wasn't sure either, so I checked, and the comma should probably land here. I think the punctuation mark order is now correct.
```suggestion
The most popular use case for the Decoupled editor is the “document editor,” similar to large editing packages such as Google Docs or Microsoft Word. It works best for creating documents, which are usually later printed or exported to PDF files.
```

**Review by gorzelinski — COMMENTED** [review]
I added some minor suggestions.

File: docs/getting-started/integrations-cdn/dotnet.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 category: cloud
 meta-title: Compatibility with .NET | CKEditor 5 documentation
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/getting-started/integrations-cdn/dotnet.md:3]: "Compatibility with .NET with CDN" sounds kinda weird. We might want to rephrase the whole title, tho.

File: docs/getting-started/setup/menubar.md [hunk 4]
```diff
@@ -35,6 +36,7 @@ The menu bar is disabled by default. To make it available in your editor, set th
 ```js
 ClassicEditor
 	.create( document.querySelector( '#editor' ), {
+		licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/getting-started/setup/menubar.md:39]: We may remove it, but shouldn't we replace it with `Other configuration here` then? In such a case, isn't it better to just keep the `licenseKey` as both are one line?

File: docs/getting-started/integrations-cdn/quick-start.md [hunk 5]
```diff
@@ -1,13 +1,14 @@
 ---
 menu-title: Vanilla JS
-meta-title: Vanilla JS CKEditor 5 installation | CKEditor 5 documentation
+meta-title: Vanilla JS CKEditor 5 Quick Start with CDN | CKEditor 5 documentation
```
[the remark this answers is not in the document]
Comment (godai78) [reply docs/getting-started/integrations-cdn/quick-start.md:3]: Wouldn't "Quick Start Installation" look abundant?

File: docs/getting-started/setup/menubar.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (gorzelinski) [reply docs/getting-started/setup/menubar.md:39]: We can keep it, I guess.

**Review by gorzelinski — APPROVED** [review]
LGTM

**Pull request merged**